ext/page_log/palm/palm_kv.c

```diff
@@ -41,15 +41,13 @@
 
 #include "palm_kv.h"
 
-/*
- * Set the size of the LMDB cache. Ideally, this should be a parameter of some kind.
- */
-static const size_t PALM_LMDB_CACHE_SIZE = 100 * 1024 * 1024;
+static const size_t MEGABYTE = 1024 * 1024;
```

Comment:
Why not `#define`? This is a constant... Same for `PALM_MAX_DBI` below.

Reply:
Funny I thought `static const` was preferred over #define (due to type checking and other advantages of it being a real symbol-table entity).  Any reason to want #define here?

Reply:
That's in C++, where `constexpr` is preferred.

Reply:
Okay I'll move to #define, (although I think the static allows it to work in headers...)